iotdb-core/datanode/src/main/java/org/apache/iotdb/db/pipe/connector/protocol/thrift/sync/IoTDBDataNodeSyncConnector.java

```diff
@@ -96,7 +97,8 @@ protected IoTDBSyncClientManager constructClient(
             trustStorePwd,
             useLeaderCache,
             loadBalanceStrategy,
-            shouldReceiverConvertOnTypeMismatch);
+            shouldReceiverConvertOnTypeMismatch,
+            loadTsFileStrategy);
```

Comment:
Consider the "schemaRegionConnector" instance, the non-dataRegion connectors shall not pass "async-load"